docs/source/keras.rst

```diff
@@ -14,8 +14,8 @@ these packages need to be installed:
 
 .. code-block:: bash
 
-   $ pip install tensorflow>=2.3.0
-   $ pip install scikeras>=0.1.8
+   $ pip install tensorflow>=2.4.0
+   $ pip install scikeras>=0.3.2
```

Comment:
Why is this required? Dask-ML will still work with the removed versions, right?

Reply:
This should work generally, but some of the syntax in this tutorial may not. I think we should either update the versions, or remove them altogether (since not specifying a version usually gets you the latest version anyway).

Reply:
Dask-ML will not work with SciKeras v0.1.7. I think that version didn't have serialization (?).

We should make a note about the versioning. "The example below uses X. The usage with lower versions may be different than this example."

Reply:
Wasn't there also some issue about serialization of stateful optimizers like Adam?

Reply:
That's a good reason to require SciKeras v0.3.0.